app/scripts/vendor/coseme-client/client.js

```diff
@@ -241,33 +241,49 @@
 
       function onSubjectUpdated(senderJID, timestamp, messageId,
                                 subject, displayName, author) {
-        fire('notification', pub,
-          newNotification('group-subject', messageId, senderJID,
-                          { subject: subject }, displayName, author)
-        );
+        fire('notification', pub, newNotification(
+          'group-subject',
+          messageId,
+          senderJID,
+          { subject: subject },
+          timestamp,
```

Comment:
Just to not break anything, could we put `timestamp` as the latest parameter? Just in case :)